plugins/generator-addon-1.21.x/addon-1.21.x/templates/item.json.ftl

```diff
@@ -13,13 +13,12 @@
       "minecraft:max_stack_size": ${data.stackSize},
       "minecraft:max_damage": ${data.maxDurability},
       <#if data.isFood>
-      "minecraft:use_duration": ${data.useDuration},
+      "minecraft:use_duration": ${data.useDuration * 20},

```

Comment:
We usually use ticks for this, so it may make sense to keep doing so. Or is in 1.21.5 template this in seconds?

Reply:
Ah sory, missed your comment above :D